.github/workflows/ci_pipe.yml

```diff
@@ -230,7 +230,7 @@ jobs:
       matrix:
         arch: ["amd64", "arm64"]
         include:
-          - runner: linux-amd64-gpu-v100-latest-1
+          - runner: linux-amd64-gpu-l4-latest-1
```

Comment:
This was updated here because I forgot to do it on a prior PR that updated the runners to no longer use the v100